src/App.js

```diff
@@ -131,7 +131,7 @@ function App() {
             <Route path="*" element={<PageNotFound />} />
           </Routes>
         </main>
-        <Footer />
+        <Footer handleLogout={handleLogout} user={user} userId={userData?.id} />
```

Comment:
Is the question mark a typo?

Reply:
No. It is shorthand used for chaining

Reply:
Right, yet in this case it's essentially pointless so I was curious if you wanted to keep it.